netbsd/NetBSDProcessList.c

```diff
@@ -366,29 +402,31 @@ static void NetBSDProcessList_scanCPUTime(NetBSDProcessList* this) {
    }
 
    for (int i = 0; i < CPUSTATES; i++) {
-      avg[i] /= this->super.cpuCount;
+      avg[i] /= this->super.activeCPUs;
    }
 
-   kernelCPUTimesToHtop(avg, this->cpus);
+   kernelCPUTimesToHtop(avg, &this->cpuData[0]);
 }
 
 static void NetBSDProcessList_scanCPUFrequency(NetBSDProcessList* this) {
-   unsigned int cpus = this->super.cpuCount;
+   unsigned int cpus = this->super.existingCPUs;
    bool match = false;
    char name[64];
    int freq = 0;
    size_t freqSize;
 
-   for (unsigned int i = 0; i <= cpus; i++) {
-      this->cpus[i].frequency = NAN;
+   for (unsigned int i = 0; i < cpus; i++) {
+      CPUData* cpu = &this->cpuData[i + 1];
+      cpu->frequency = NAN;
```

Comment:
Aliasing doesn't make much sense here. Actually makes it less readable IMHO.

Reply:
Should be fixed in the latest commit.